hack/make-rules/crossbuildimage.sh

```diff
@@ -82,6 +82,14 @@ function build_multi_arch_images() {
     DOCKERFILE_PATH="$(get_dockerfile_by_target ${arg})"
 
     set -x
+    HOST_ARCH=$(uname -m)
+    if [ x"${HOST_ARCH}" == x"aarch64" ]; then
+        echo "Building arm64 image natively"
+        exit
+    fi
+    # Enable cross-platform builds https://github.com/multiarch/qemu-user-static
+    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
do we need this line? docker buildx will automatically build cross-platform images even though without this line?

Reply:
In some env, we need to run this cmd to fix the issue.

@zhu733756 Can we add the two issues to the comments? 
``
If there's any issues when using buildx, can refer to the issue below
# https://github.com/docker/buildx/issues/495
# https://github.com/multiarch/qemu-user-static/issues/100
``

Reply:
OK. I will comment it out.

Reply:
LGTM